doc/nrf/test_and_optimize/testing.rst

```diff
@@ -89,31 +89,25 @@ To run RTT on your platform, complete the following steps:
 
 #. Click :guilabel:`OK` to view the logging output from the device.
 
-.. _lte_connect:
+.. _serial_terminal_connect:
 
-How to connect with LTE Link Monitor
-************************************
+How to connect with Serial Terminal
+***********************************
 
-To connect to nRF9160-based kits (for example, the nRF9160 DK or Nordic Thingy:91), you can also use `LTE Link Monitor`_, which is implemented in `nRF Connect for Desktop`_.
+To connect to nRF91 Series devices (for example, the nRF9161 DK, the nRF9160 DK or Thingy:91), you can also use the `nRF Connect Serial Terminal`_ app, which is implemented in `nRF Connect for Desktop`_.
```

Comment:
```suggestion
To connect to nRF91 Series devices (for example, the nRF9161 DK, the nRF9160 DK or Thingy:91), you can use the `nRF Connect Serial Terminal`_ app, which is part of in `nRF Connect for Desktop`_.
```
Doesn't the nRF Connect Serial Terminal app work for all nRF series? Why is only nRF91 mentioned here specifically?